src/Microsoft.AspNet.SignalR.Core/TaskAsyncHelper.cs

```diff
@@ -710,7 +710,7 @@ public static Task Delay(TimeSpan timeOut)
 #else
             var tcs = new TaskCompletionSource<object>();
 
-            var timer = new Timer(tcs.SetResult,
+            var timer = new Timer(tcs.TrySetResult,
```

Comment:
Fix compile error

Reply:
Ah, it's because `Try` methods return a bool. Weird. Fixing.